playbooks/graphql-schema-design.md

```diff
@@ -123,6 +135,28 @@ e.g. an `artworks` field _and_ an `artworks_connection`, simply choose one form
 - In rare cases an immediate list may be used, but this should only be done in cases where the list has few entries
   and doesn’t require pagination, i.e. fetch _all_ entries at once.
 
+## ID fields
+
+An ID field that refers to e.g. a database ID should be called something like `internalID`, it MUST _never_ be called
```

Comment:
Interesting! So does this mean there won't be the official support of specifying an `__id` as an `idField`? I thought (at least in MP) we adhere to that, so therefore continuing to support `id` as a database ID or slug was ok.

But, also `internalID` sounds totally fine to me as being prescribed, so wondering if I'm missing some active movement around next steps with the Relay fork.

Reply:
We’ll be having a larger discussion in the platform practice after my vacation, but it does seem ideal to get off of the fork sooner rather than later–as we won’t know how long any official support, if any, may still take.

Reply:
@mzikherman some graphql implementations now give a [deprecation warning](https://github.com/graphql/graphql-js/blob/77c36183ff46d7d8853f8d6fe1051ebd59d89392/src/utilities/assertValidName.js#L37) if you explicitly express fields with `__`. It seems those are reserved for internal usage and code relying on it may break in future releases. 

Beyond that, the global object identification spec requires that the `id` field be globally unique (and subsequently traceable to its origin). 

Basically just means that how metaphysics is currently running requires that we maintain our relay fork (and potentially forks of graphql server implementations if they start reserving `__` in a breaking way). To get away from that we have to both change how `id` is used and remove usage of `__id`.